spec/lib/logstash/bundler_spec.rb

```diff
@@ -27,4 +27,96 @@
       expect(exception.message).to eq("baz")
     end
   end
+  
+  context 'when invoking bundler' do
+    original_stderr = $stderr
```

Comment:
makes sense to have this in to before block as the one you have in the after for restoring? might be just before line 40?